pkg/psa/psa.go

```diff
@@ -43,3 +47,7 @@ func IsNamespacePrivileged(namespace *k8sv1.Namespace) bool {
 	enforceLevel, labelExist := namespace.Labels[PSALabel]
 	return labelExist && enforceLevel == "privileged"
 }
+
+func IsVMIPrivileged(vmi *virtv1.VirtualMachineInstance) bool {
```

Comment:
I'd change the function name to something like `requiresVMIPrivileged`
Is PSA only required on openshift or also on vanilla k8s?

Reply:
actually, what do you think about `requiresPrivilegedNamespace`?

PSA is also present on vanilla k8s.